### PR TITLE
Add ocean

### PR DESCRIPTION
**Pull request by Rondeau79**

### CODE REVIEW

**Review by gwolf2u — CHANGES_REQUESTED** [review]
Please use createjson.sh 
Don't create ocean.json file manually
That's what the script is for
Read readme file

**Comment by Rondeau79** [comment]
I did use that script. The readme instructions are ambiguous, this whole process needs to be broken down. There is no way to get into the initial support folder from Linux terminal.
Folder cannot have a space between words. I don't get where this went wrong.

**Comment by gwolf2u** [comment]
No you didn't use script
If you had, more data would be present

Anyway, cd to space dir
cd "initial support"

**Comment by Rondeau79** [comment]
bash: cd: initial: No such file or directory

**Comment by gwolf2u** [comment]
Read on google
How to cd to space dir linux

File: ocean.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+                        "oem": "Motorola",
+                        "device": "Moto G7 Power",
+                        "filename": "crDroidAndroid-10.0-20200322-ocean-v6.4.zip",
+                        "download": "https://sourceforge.net/projects/crdroid/files/ocean/6.x/crDroidAndroid-10.0-20200322-ocean-v6.4.zip/download",
+                        "timestamp": 1584841657,
+                        "md5": "2a53c92d1aba2a8869d191e35fb15f5a",
+                        "size": 602247619,
+                        "version": "6.4",
+                        "buildtype": "Beta",
+                        "forum": "https://link",
```
Comment (gwolf2u) [inline ocean.json]: Forum link is mandatory
Rest leave empty if unused (remove https://link)

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Pull request merged**